blockchainetl/streaming/streamer.py

```diff
@@ -33,7 +33,7 @@ class Streamer:
     def __init__(
             self,
             blockchain_streamer_adapter=StreamerAdapterStub(),
-            last_synced_block_file='last_synced_block.txt',
+            last_synced_block_file='sync/last_synced_block.txt',
```

Comment:
I wouldn't change that, it's a default so when you open PR probably will be rejected.